From 5b0c6006ef0b9c2c3a8770921b21965eba0e4372 Mon Sep 17 00:00:00 2001
From: Gerrit Pape <pape@smarden.org>
Date: Wed, 19 Nov 2008 20:00:53 +0000
Subject: [PATCH] dbclient.1: dbclient uses compression if compiled with zlib support

On Fri, Sep 26, 2008 at 03:40:57PM +0200, Luca Capello wrote:
> On Tue, 23 Sep 2008 16:24:25 +0200, Matt Johnston wrote:
> > On Wed, Aug 20, 2008 at 07:43:58PM +0200, Luca Capello wrote:
> >> Could you please clarify if dbclient really support compression or not?
> >> If yes, could you please explain in the manpage how to enable it?
> >
> > As long as Dropbear was compiled with zlib support (which I
> > assume is the case for Debian)
>
> It seems so, at least the source build-depends on libz-dev and the
> binary depends on zlib1g.
>
> > dbclient will always use compression if the server supports it. There
> > isn't any option to disable it at the moment.
>
> Is there a way to test that compression is used?  I'm sorry, but I'm
> really not an expert about these stuff.
>
> Could the following patch against version 0.51 be applied to the
> dbclient manpage?
---
 dbclient.1 |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/dbclient.1 b/dbclient.1
index 96e6c5c..99f999b 100644
--- a/dbclient.1
+++ b/dbclient.1
@@ -21,6 +21,9 @@ dbclient \- lightweight SSH2 client
 .B dbclient
 is a SSH 2 client designed to be small enough to be used in small memory
 environments, while still being functional and secure enough for general use.
+.P
+If compiled with zlib support and if the server supports it, dbclient will
+always use compression.
 .SH OPTIONS
 .TP
 .B \-p \fIport
-- 
1.6.0

